packages/web/src/components/spa/HackAUBG/JudgesSection/judges_section.css

```diff
@@ -0,0 +1,37 @@
+.judges-container {
+    box-sizing: border-box;
+    margin: 0;
+    font-family: 'Roboto';
+    padding: 3% 0%;
+    background-color: #222222;
+}
+
+.judges-header {
+    text-align: center;
+    color: #19b700;
+    padding: 1% 0;
+}
+
+.judges-picture {
```

Comment:
Lose this one. If I recall correctly this one was randomly placed as an unneeded holder for the cards. Refer to the Jobs section PR of Aleks, we've resolved the same issue there.